requirements/test_requirements.in

```diff
@@ -18,3 +18,20 @@ nbconvert~=5.6
 urllib3>=1.24.2,<2.0
 pytest-benchmark~=3.2
 mock~=3.0
+flakehell~=0.7.0
+flake8-broken-line~=0.3
+flake8-bugbear~=20.11
+flake8-builtins~=1.5
+flake8-commas~=2.0
+flake8-comprehensions~=3.3
+darglint~=1.5
+flake8-debugger~=4.0
+flake8-docstrings~=1.5
+flake8-eradicate~=1.0
+flake8-isort~=4.0
+flake8-mutable~=1.2
+flake8-string-format~=0.3
+mccabe~=0.6
+pep8-naming~=0.11.1
+pycodestyle~=2.6
+pyflakes~=2.2
```

Comment:
versions should be low and freezed cause of the massive conflicts in the requirements files.